.github/workflows/ci-california-api.yml

```
@@ -1,4 +1,4 @@
-name: CI - FastAPI Template
+name: CI - California API Run Checks
```

```suggestion
name: CI - Example: California API Run Checks
```

If we're going to be renaming the files and the workflows I'd argue for something like: 

- File: `ci-example-california.yaml`
- Name: `CI - Example: California API Checks`

I'll leave comments on the others but I like the pseudo pattern you landed on, and might suggest formalizing it in this way:

- For PR checks:
  - File: `ci-[directory]-[descriptive-name].yaml`
  - Name: `CI - [Directory]: [Descriptive name] Run Checks`
- For deploy workflows:
  - File: `cd-deploy-[directory]-[descriptive-name].yaml`
  - Name: `CD - Deploy [Directory] [Descriptive name] to [Deploy Target]`

---

That way the CI checks for a given sub-directory will be grouped in the  file list _and_ in the GitHub actions UI